Tests/SentryTests/Performance/SentryTracerTests.swift

```diff
@@ -810,13 +810,6 @@ class SentryTracerTests: XCTestCase {
     }
 #endif
     
-    func testSetExtra_ForwardsToSetData() {
```

Comment:
`m`: We didn't remove the function yet. I would still keep the tests.

Reply:
Our linter does not allow use of deprecated API, and there is no warning suppressing declarative for Swift.

Reply:
Well, well, well. 
If we mark a function as deprecated we can use other deprecated attributes inside of it.
A nice way around for tests

Reply:
Is SwiftLint complaining? You can disable rules see https://github.com/realm/SwiftLint#disable-rules-in-code. Otherwise smart hack 😀👏